lib/rubocop/cop/lint/no_env.rb

```diff
@@ -15,7 +15,9 @@ module Lint
       #
       #   # good
       #   GetEnv.fetch(...)
-      class NoENV < Cop
+      class NoENV < Base
```

Comment:
This fixes the warnin saying that we should inherit from `Base` instead of `Cop`